### ISSUE TITLE
[BUG] Unable to use a directory on a windows network share as the target

### ISSUE THREAD

**Issue by pcanning** [rdiff-backup/rdiff-backup#353]
## Bug summary

I originally discovered this problem using the Cygwin port of rdiff-backup as I described in the second half of https://cygwin.com/pipermail/cygwin/2020-May/244713.html.  I later discovered the native windows version of rdiff-backup at https://github.com/rdiff-backup/rdiff-backup/releases/tag/v2.0.0 and reproduced the same issue using the native windows executable.

When I specify a target directory that is on a network share (e.g. //nas4free/test-backup/home.rdiff-backup or \\nas4free\test-backup\home.rdiff-backup) rdiff-backup reports an error saying that it is unable to create the target directory.  The error message shows the target directory using forward slashes, but with only a single slash (/) at the beginning.  For example:
Fatal Error: Unable to create directory /nas4free/test-backup/home.rdiff-backup

## Version, Python, Operating System

Call `rdiff-backup -v9` and paste the output between the backquotes below, repeat for each environment impacted:

```
2020-05-07 22:40:27.074493 -0700  <CLIENT-18416>  Using rdiff-backup version 2.0.0
2020-05-07 22:40:27.074493 -0700  <CLIENT-18416>        with cpython C:\Users\pcanning\Downloads\rdiff-backup-2.0.0.win32exe\rdiff-backup-2.0.0\rdiff-backup.exe version 3.7.5
2020-05-07 22:40:27.075462 -0700  <CLIENT-18416>        on Windows-10-10.0.18362, fs encoding utf-8
2020-05-07 22:40:27.075462 -0700  <CLIENT-18416>  Fatal Error: No arguments given
See the rdiff-backup manual page for more information.
```

## rdiff-backup call

How did you call rdiff-backup, with which parameter:

```
.\rdiff-backup.exe C:\Users\username //nas4free/test-backup/home.rdiff-backup
or
.\rdiff-backup.exe C:\Users\username \\nas4free\test-backup\home.rdiff-backup
```

## What happened and what did you expect?
rdiff-backup did not create the directory home.rdiff-backup inside the network share //nas4free/test-backup, and produced the error message:
Fatal Error: Unable to create directory /nas4free/test-backup/home.rdiff-backup

I expected it to create the directory home.rdiff-backup inside the network share //nas4free/test-backup and backup the source directory into that directory.

## More information

The next to last paragraph of section 4.13 of POSIX (https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap04.html#tag_04_13) says "If a pathname begins with two successive <slash> characters, the first component following the leading <slash> characters may be interpreted in an implementation-defined manner, although more than two leading <slash> characters shall be treated as a single <slash> character.".  I would say it is subject to interpretation whether converting two leading <slash> characters into one leading <slash> character is legal, but it clearly doesn't work correctly on Windows. 

I was able to trace through the code in pdb, to narrow down the issue.  It appears that the two leading <slash> characters are converted into one leading <slash> character at in the first statement in rpath.Rpath.normalize
        newpath = self.path_join(
            b'', *[x for x in self.path.split(b"/") if x and x != b"."])
path has the two leading <slash> characters , but newpath  only has one leading <slash> character.

If you can, please repeat the action leading to the error, adding the option `-v9` and attach the output to this bug report.

rdiff-backup-2.0.0> .\rdiff-backup.exe -v9 C:\Users\pcanning //nas4free/test-backup/home.rdiff-backup
2020-05-07 22:45:26.940982 -0700  <CLIENT-14716>  Using rdiff-backup version 2.0.0
2020-05-07 22:45:26.940982 -0700  <CLIENT-14716>        with cpython C:\Users\pcanning\Downloads\rdiff-backup-2.0.0.win32exe\rdiff-backup-2.0.0\rdiff-backup.exe version 3.7.5
2020-05-07 22:45:26.940982 -0700  <CLIENT-14716>        on Windows-10-10.0.18362, fs encoding utf-8
2020-05-07 22:45:28.105392 -0700  <CLIENT-14716>  Making directory /nas4free/test-backup/home.rdiff-backup
2020-05-07 22:45:28.105728 -0700  <CLIENT-14716>  Fatal Error: Unable to create directory /nas4free/test-backup/home.rdiff-backup

**Comment by ericzolf** [comment]
I guess, this is a duplicate of #337 and has been closed with version v2.0.1rc0, available from https://github.com/rdiff-backup/rdiff-backup/releases

**Comment by pcanning** [comment]
Thanks Eric.  I did scan through the issues looking for an existing one, but I guess I only looked at the open issues.  I have verified that this issue is resolved in 2.0.1rc0, so I'll close this issue.
